source/features/list-prs-for-file.tsx

```diff
@@ -106,9 +106,8 @@ const getPrsByFile = cache.function('files-with-prs', async (): Promise<Record<s
 });
 
 async function getCurrentPath(): Promise<string> {
-	// `[aria-label="Copy path"]` on blob page, `#blob-edit-path` on edit page
-	const element = await elementReady('[aria-label="Copy path"], #blob-edit-path');
-	return element!.getAttribute('value')!;
+	const viewDataJson = (await elementReady('[data-target="react-app.embeddedData"]'))!.textContent!.trim();
+	return JSON.parse(viewDataJson).payload.path;
```

Comment:
@yakov116 Do you remember why we're not using location.href instead?

Reply:
Its here since the feature was created https://github.com/refined-github/refined-github/pull/2197 🤷

Reply:
Indeed it's present in the very first commit and we never reviewed that. 

@Pwuts let's use `new GitHubURL(location.href).filePath` instead of this getter